nan.h

```diff
@@ -940,11 +943,31 @@ class Utf8String {
  private:
   NAN_DISALLOW_ASSIGN_COPY_MOVE(Utf8String)
 
+ protected:
   int length_;
   char *str_;
   char str_st_[1024];
 };
 
+# ifdef __MVS__
+class EbcdicString : public Utf8String {
```

Comment:
It seems wrong to have this extend `Utf8String`. It should be a sibling based on `WriteOneByteString`.